ui/webui/src/components/time/DateAndTime.jsx

```diff
@@ -0,0 +1,505 @@
+                                      isOpen={regionsOpen}
+                                      selections={timezoneRegion}
+                                      onToggle={(_, isOpen) => setRegionsOpen(isOpen)}
+                                      onSelect={(event, value) => handleRegionChange(value)}
+                                    >
+                                        {Object.keys(timezones).map(region => <SelectOption key={region} value={region}>{region}</SelectOption>)}
+                                    </Select>
+                                </FormGroup>
+                            </GridItem>
+                            <GridItem span={6}>
+                                <FormGroup
+                                  label={_("City")}
+                                  isRequired
+                                  fieldId="time-date-edit-city"
+                                >
+                                    <Select
+                                      id={idPrefix + "select-city"}
+                                      variant={SelectVariant.typeahead}
+                                      placeholderText={_("Select city")}
+                                      isOpen={citiesOpen}
+                                      selections={timezoneCity}
+                                      onToggle={(_, isOpen) => setCitiesOpen(isOpen)}
+                                      onSelect={(event, value) => handleCityChange(value)}
+                                    >
+                                        {timezoneRegion
+                                            ? timezones[timezoneRegion].map(city =>
+                                                <SelectOption
+                                                  key={city}
+                                                  value={city}
+                                                >
+                                                    {city.replaceAll("_", " ")}
+                                                </SelectOption>)
+                                            : []}
+                                    </Select>
+                                </FormGroup>
+                            </GridItem>
+                        </Grid>
+                    </FormSection>
+                    <FormSection title={_("Date and time")}>
+                        <Grid>
+                            <GridItem span={4}>
+                                <FormGroup
+                                  label={_("Automatic time")}
+                                  labelIcon={
+                                      <Popover
+                                        bodyContent={_(
+                                            "To edit date and time, turn off automatic time."
+                                        )}
+                                      >
+                                          <Icon iconSize="sm">
```

Comment:
Please copy the info code from the form example. This code you wrote - looks correct but it produces some really small icon with wrong color.